.github/workflows/coverity.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       DEBIAN_FRONTEND: noninteractive
       TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
 
+    # probably no TOKEN if it was a forked repo, then just cancel the process
+    if: ${{ !github.event.repository.fork }}
```

Comment:
Is there a document about this variable?

Reply:
perhaps, but i am not sure if there was a doc for this.